webpack/components/extensions/HostDetails/DetailsTabCards/SystemPropertiesCardExtensions.js

```diff
@@ -35,6 +36,71 @@ SystemPropertiesCardSubscription.defaultProps = {
   hostDetails: {},
 };
 
+export const SystemPropertiesCardVirtualization = ({ hostDetails }) => {
+  if (!hostDetails?.subscription_facet_attributes) return null;
+
+  const {
+    virtual_guests: virtualGuests,
+    hypervisor,
+    registered_through: registeredThrough,
+    virtual_host: virtualHost,
+  } = hostDetails.subscription_facet_attributes;
+  const virtualGuestIds = virtualGuests.map(guest => `name = ${guest.name}`).join(' or ');
```

Comment:
You can use scoped_search's "IN" operator to make this a bit cleaner - like 

```
name ^ (host1.example.com, host2.example.com)
```

```suggestion
  const virtualGuestIds = "name ^ (${virtualGuests.map(guest => guest.name).join(', ')}`;
```

https://github.com/wvanbergen/scoped_search/wiki/query-language